packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/NodeHud.tsx

```diff
@@ -174,14 +170,41 @@ export default function NodeHud({
 }: NodeHudProps) {
   const hintPosition = rect.y > HUD_HEIGHT ? HINT_POSITION_TOP : HINT_POSITION_BOTTOM;
 
+  const interactiveNodeClipPath = React.useMemo(
+    () =>
+      isInteractive && selectedNodeRect
```

Comment:
The clip path is a polygon around the selection rectangle, as it's not possible do define inversed clip paths (would have been useful).